config/webpack.config.js

```diff
@@ -119,7 +119,14 @@ var config = {
       }
     }),
     new webpack.optimize.CommonsChunkPlugin({
-      name: 'vendor'
+      name: 'vendor',
+      minChunks(module) {
+        if (module.resource && (/^.*\.(css|scss)$/).test(module.resource)) {
+          return false;
+        }
+
+        return module.context && module.context.includes("node_modules");
```

Comment:
won't this also catch plugin dependencies? e.g. if a plugin add a package, this will also increase the vendor file?